### PR TITLE
Made Annual button  default and Station button to be last in Climatic Summaries dialog

### LINKED ISSUE #7479: Improving Climatic Summary dialog by making Annual button to be the default
During KMD workshop, we found that `Annual` button was the one used mostly, we can make `Annual`  to be the default and `Station` to be the last button

image

### PR DESCRIPTION
**Pull request by Vitalis95**
Fixes #7479 
@rdstern @N-thony @lloyddewit , this PR makes `Annual` button default and `Station` to be the last button

### CODE REVIEW

**Review by lloyddewit — DISMISSED** [review]

**Comment by lloyddewit** [comment]
@rdstern if you can test/approve, then we can merge, thanks

**Review by N-thony — DISMISSED** [review]

File: instat/dlgClimaticSummary.vb [hunk 1]
```diff
@@ -62,17 +62,12 @@ Public Class dlgClimaticSummary
...
-        ucrPnlAnnualWithin.AddParameterValuesCondition(rdoAnnualWithinYear, "checked", "annual_within_year")
-        ucrPnlAnnualWithin.AddParameterValuesCondition(rdoWithinYear, "checked", "within_year")
-        ucrPnlAnnualWithin.AddParameterValuesCondition(rdoStation, "checked", "station")
-        ucrPnlAnnualWithin.AddParameterValuesCondition(rdoDaily, "checked", "daily")
+        ucrPnlAnnualWithin.SetParameter(New RParameter("checked", 0))
+        ucrPnlAnnualWithin.AddRadioButton(rdoAnnual, "annual")
+        ucrPnlAnnualWithin.AddRadioButton(rdoAnnualWithinYear, "annual_within_year")
+        ucrPnlAnnualWithin.AddRadioButton(rdoWithinYear, "within_year")
+        ucrPnlAnnualWithin.AddRadioButton(rdoStation, "station")
+        ucrPnlAnnualWithin.AddRadioButton(rdoDaily, "daily")
```
Comment (shadrackkibet) [inline instat/dlgClimaticSummary.vb:70]: @Vitalis95 I removed the manual addition and did this via setRcode. Both ways "works" but it is much more elegant to do it by setting a parameter. Does this make sense to you?

**Review by shadrackkibet — APPROVED** [review]
@rdstern if you test and re-approve then we can merge. Thanks

**Review by rdstern — APPROVED** [review]
Looks good.  I feel we are really getting there!

**Pull request merged**